server/@types/express/index.d.ts

```diff
@@ -25,6 +26,7 @@ export declare global {
       verified?: boolean
       id: string
       logout(done: (err: unknown) => void): void
+      services?: Services
```

Comment:
could you pls remove the `services` property on requests? i am pretty sure you'd already removed the request handler that put it there.

i think the separation of requests and shared services will be easier to reason about if services are passed to middleware _directly_, not via requests. request flows already have various instantiated api classes via `res.locals.apis`

Reply:
Oh yeah missed this, I will go back and remove it